WordPress/Classes/Categories/NSURLRequest+Helpers.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface NSURLRequest (Helpers)
```

Comment:
You're indeed right Diego, and you also do have a great taste. Let's go for it, thank you!